sql/create.go

```diff
@@ -121,11 +120,14 @@ func (p *planner) CreateIndex(n *parser.CreateIndex) (planNode, error) {
 	var b client.Batch
 	b.Put(MakeDescMetadataKey(tableDesc.GetID()), tableDesc)
 
+	// dummy write so indexing works later
```

Comment:
Can you expand on this comment? Why would you need to index into `writes` if there were no writes.